pri_pol_evaluator.py

```diff
@@ -0,0 +1,83 @@
+from privacy_policy_evaluator import paragraphing, commands, preprocessing, helpers, correlation, wordscoring, topic_grouper

```

Comment:
clarify authorship of code. this is important for Leah in the future and ensuring we have code that she can verify authorship of